Mistake a typing of EthereumProvider.send

#### Expected behavior
The following code is valid. 
```typescript
import Web3 from 'web3'
const web3 = new Web3('')
web3.currentProvider.send('', [])
```

Because the interface of `send` conforms to [EIP-1193](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-1193.md).

#### Actual behavior
But I got an error when running `tsc --noEmit` command.

#### Steps to reproduce the behavior

#### index.ts
```typescript:index.ts
import Web3 from 'web3'
const web3 = new Web3('')
web3.currentProvider.send('', [])
```

#### tsconfig.json
```json
{
  "compilerOptions": {
      "module": "commonjs",
      "lib": ["es6"],
      "target": "es6",
      "noImplicitAny": true,
      "noImplicitThis": true,
      "strictNullChecks": true,
      "strictFunctionTypes": true,
      "noEmit": true,
      "allowSyntheticDefaultImports": true,
      "baseUrl": ".",
      "paths": { 
          "web3-providers": ["."] 
      }
  }
}
```

#### command
```bash
$ yarn tsc --project tsconfig.json --noEmit
```

#### Error Logs
```
index.ts:3:1 - error TS2349: Cannot invoke an expression whose type lacks a call signature. Type '((method: string, parameters: any[]) => Promise<object>) | ((method: string, parameters: any[]) => Promise<object>) | ((method: string, callback: () => void) => Promise<object>)' has no compatible call signatures.

5 web3.currentProvider.send('', [])
  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

#### Versions
- npm: 5.6.0
- node: 8.11.3
- web3.js: 1.0.0-beta.41
- OS: macOS High Sierra

--- a/packages/web3-providers/types/index.d.ts
+++ b/packages/web3-providers/types/index.d.ts
@@ -112,7 +112,7 @@ export class EthereumProvider {
     host: string;
     registerEventListeners(): void;
 
-    send(method: string, callback: () => void): Promise<object>;
+    send(method: string, parameters: any[]): Promise<object>;
 
     sendBatch(methods: AbstractMethod[], moduleInstance: AbstractWeb3Module): Promise<object[]>;
 

